src/Logging.AzureAppServices/src/AzureAppServicesLoggerFactoryExtensions.cs

```diff
@@ -21,15 +22,30 @@ public static class AzureAppServicesLoggerFactoryExtensions
         /// Adds an Azure Web Apps diagnostics logger.
         /// </summary>
         /// <param name="builder">The extension method argument</param>
+        /// <returns></returns>
         public static ILoggingBuilder AddAzureWebAppDiagnostics(this ILoggingBuilder builder)
         {
             var context = WebAppContext.Default;
 
             // Only add the provider if we're in Azure WebApp. That cannot change once the apps started
-            return AddAzureWebAppDiagnostics(builder, context);
+            return AddAzureWebAppDiagnostics(builder, context, _ => { });
         }
 
-        internal static ILoggingBuilder AddAzureWebAppDiagnostics(this ILoggingBuilder builder, IWebAppContext context)
+        /// <summary>
+        /// Adds an Azure Web Apps diagnostics logger. With CustomPrefix the Blob output will be in flat file strucure
+        /// </summary>
+        /// <param name="builder">The extension method argument</param>
+        /// <param name="customPrefix"></param>
```

Comment:
?

Reply:
Are xml docs turned off for this project

Reply:
This is super odd, I have no idea why API docs check is not running. As far as I can tell from the binlog, it should be enabled.

Reply:
@dotnet/aspnet-build Any ideas why API doc checks aren't triggering for this project? I looked at the preprocessor and binlog and the errors aren't explicitly disabled. I'm at a loss to explain why other projects have API checks but this project doesn't.

Reply:
I'll open an issue to track this discussion since I'm concerned about our blindspots. Won't hold up this PR though.

Reply:
Don't open an issue. `$(GenerateDocumentationFile)` is an opt-in setting

Reply:
It's enabled now.